server/data/dbConfig.js

```diff
@@ -1,6 +1,6 @@
 const knex = require("knex");
 const config = require("../knexfile.js");
 
-const dbEnv = process.env.DB_ENV || "production";
+const dbEnv = process.env.DB_ENV || "staging";
```

Comment:
Did you change this on purpose? I know that having it set to `staging` was causing problems a few weeks ago. I would check with @iepoch before this is merged.